api/profile/profileRouter.js

```diff
@@ -381,4 +376,24 @@ router.get('/match/:id', authRequired, (req, res, next) => {
     });
 });
 
+router.post('/mentor-information/:profile_id', (req, res, next) => {
+  const { city, accepting_new_mentees, profile_id } = req.body;
+  axios.post(`${process.env.DS_API_URL}/read/mentor`)
+  .then((res) => {
+    res.data.result.map((results) => {
+      let profile_id = results.profile_id
+      let city = [results.city, results.state]
+      let availability = results.accepting_new_mentees
+      availability === false
+        ? availability = 'No'
```

Comment:
the string value of a 'boolean' should probably be handled by the UI or FE developers.